classes/helpers/FrmListHelper.php

```
@@ -824,12 +824,11 @@ protected function get_sortable_columns() {
 	 * @return string Name of the default primary column, in this case, an empty string.
 	 */
 	protected function get_default_primary_column_name() {
-		$columns = $this->get_columns();
 		$column  = '';
 
 		// We need a primary defined so responsive views show something,
 		// So let's fall back to the first non-checkbox column.
-		foreach ( $columns as $col => $column_name ) {
+		foreach ( $this->get_columns() as $col => $column_name ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$column_name'. (undefined)

(UnusedLocalVariable)